internal/service/ec2/transitgateway_route_table_association_test.go

```diff
@@ -175,6 +231,9 @@ func testAccCheckTransitGatewayRouteTableAssociationDestroy(ctx context.Context)
 	}
 }
 
+// testAccCheckRouteTableAssociationNotRecreated function, as is typical, cannot work here because
+// the resource ID will be the same, even if the association is recreated.
```

Comment:
Could this comment move up into the "not recreated" test case? I wonder if it'd get lost down here for a future reviewer wondering why this typical check is missing.

Reply:
Good call